user_docs/en/changes.t2t

```diff
@@ -22,21 +22,21 @@ What's New in NVDA
 
 
 == Changes ==
-- The list of available languages in the General Settings dialog is now sorted based on language names instead of ISO 639 codes. (#7284)
-- Add default gestures for alt shift tab and windows tab with all supported Freedom Scientific braille displays. (#7387)
+- The list of available languages in NVDA's General Settings is now sorted based on language names instead of ISO 639 codes. (#7284)
+- Added default gestures for Alt+Shift+Tab and Windows+Tab with all supported Freedom Scientific braille displays. (#7387)
 - For ALVA BC680 and protocol converter displays, it is now possible to assign different functions to the left and right smart pad, thumb and etouch keys. (#8230)
 - For ALVA BC6 displays, the key combination sp2+sp3 will now announce the current date and time, whereas sp1+sp2 emulates the Windows key. (#8230)
-- The user is now asked once on NVDA start-up if they are happy sending usage statistics to NV Access when automatically checking for updates. (#8217)
+- After installing 2018.3, the user is asked once when NVDA starts if they are happy sending usage statistics to NV Access when automatically checking for updates. (#8217)
 - When checking for updates, if the user has agreed to allow sending usage statistics to NV Access, NVDA will now send the name of the current synth driver and braille display in use, to aide in better prioritization for future work on these drivers. (#8217)
 - Updated liblouis braille translator to version 3.6.0. (#8365)
-- Updated the path to the correct russian eight-dots Braille table. (#8446)
-- Updated eSpeak-ng to 1.49.3dev commit 910f4c2 (#8561)
+- Updated the path to the correct Russian eight-dots Braille table. (#8446)
+- Updated eSpeak-ng to 1.49.3dev commit 910f4c2. (#8561)
 
 
 == Bug Fixes ==
 - Accessible labels for controls in Google Chrome are now more readily reported in browse mode when the label does not appear as content itself. (#4773)
-- Notifications are now supported in Zoom. For example, this includes mute/unmute status, and incoming messages.(#7754)
-- Switching braille context presentation when in browse mode no longer causes braille output to stop following. (#7741)
+- Notifications are now supported in Zoom. For example, this includes mute/unmute status, and incoming messages. (#7754)
+- Switching braille context presentation when in browse mode no longer causes braille output to stop following browse mode cursor. (#7741)

```

Comment:
This new information is redundant imo. If NVDA is in browse mode and braille is supposed to follow, I'd say it is obvious that it should follow the browse mode cursor.